docs/docs/babel.md

```diff
@@ -22,48 +22,26 @@ browsers.
 ## How to use a custom .babelrc file
 
 Gatsby ships with a default .babelrc setup that should work for most sites. If you'd like
-to add custom Babel presets or plugins, we recommend copying our default .babelrc below
-to the root of your site and modifying it per your needs.
+to add custom Babel presets or plugins, you can import our preset and overwrite the target option.
```

Comment:
does "our preset" mean `babel-preset-gatsby`? If so, it would be clearer to just say that instead of "our preset."